src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  &__container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+    width: 944px;
```

Comment:
I recomend use width calculation not number (width cards for 4 cards, margins and paddings)

Reply:
Thx!
Done